cmd/podman/containers/ps.go

```diff
@@ -47,7 +48,9 @@ func init() {
 		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
 		Command: psCommand,
 	})
-	flags := psCommand.Flags()
+	listFlagSet(psCommand.Flags())
+}
+func listFlagSet(flags *pflag.FlagSet) {
```

Comment:
a nite of a nit, it would be nice to have a blank line between the last function and the start of this function.